doc-content/enterprise-only/project-data/test-designer-create-test-template-dmn-based-proc.adoc

```diff
@@ -0,0 +1,34 @@
+[id='test-designer-create-test-template-dmn-based-proc']
+= Creating a test scenario template for DMN-based test scenarios
+
+Create a test scenario template for DMN-based scenarios by following the procedure below to validate your rules and data.
+
+.Procedure
+. In {CENTRAL}, go to *Menu* -> *Design* -> *Projects* and click the project for which you want to create the test scenario.
+. Click *Add Asset* -> *Test Scenario*.
+. Enter a *Test Scenario* name and select the appropriate *Package*. The package you select must contain all the required data objects and rule assets have been assigned or will be assigned.
+. Select *DMN* as the *Source type*.
+. Select an existing DMN asset using the *Choose DMN asset* option.
+. Click *Ok* to create and open the test scenario in the *Test Scenario* designer.
+. To map the *GIVEN* column header to a data object,
+.. Click an instance header in the *GIVEN* section.
+.. Select the data object from the *Test Editor* tab.
+.. Click *Add*.
+. To insert more properties of the data object, right-click the property header and select *Insert column right* or *Insert column left* as required.
+. To map a data object field to a property cell,
+.. Click a property cell.
+.. Select the data object field from the *Test Editor* tab.
+.. Click *Add*.
+. To map the *EXPECTED* column header to a data object,
+.. Click an instance header in the *EXPECTED* section.
+.. Select the data object from the *Test Editor* tab.
+.. Click *Add*.
+. To insert more properties of the data object, right-click the property header and select *Insert column right* or *Insert column left* as required.
+. To map a data object field to a property cell,
+.. Click a property cell.
+.. Select the data object field from the *Test Editor* tab.
+.. Click *Add*.
+
+Use the contextual menu to add or remove columns and rows as needed.
+
+After you have created and mapped both the *GIVEN* & *EXPECTED* columns to a data object & its properties, you have to define the test scenario which comes next.
```

Comment:
Minor suggestion - simplify and replace "&" with "and":

s/After you have created and mapped both the *GIVEN* & *EXPECTED* columns to a data object & its properties, you have to define the test scenario which comes next./After you create and map both the *GIVEN* and *EXPECTED* columns to a data object and its properties, you then have to define the test scenario.

Reply:
Changed as suggested.